src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadocpackage/Example2.java

```diff
@@ -0,0 +1,17 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MissingJavadocPackage"/>
+  </module>
+</module>
+*/
+// xdoc section -- start
+/*
```

Comment:
violation is not raised for the above block comment for some reason. @romani can you please provide some insights.